app/models/external_activity.rb

```diff
@@ -3,6 +3,7 @@ class ExternalActivity < ActiveRecord::Base
 
   # see https://github.com/sunspot/sunspot/blob/master/README.md
   searchable do
+    integer :id
```

Comment:
@scytacki I had to add that, otherwise the following `without` part of the query doesn't work to exclude the original material. (I.e. top related match will always be the same material as the original.)

<pre>
            search = Sunspot.search(Search::SearchableModels) do

                fulltext "*" do
                    boost(4.0) { with(:subject_areas, tags['subject_areas']) }
                    boost(2.0) { with(:grade_levels, tags['grade_levels']) }
                    boost(1.0) { with(:project_names, projects) }
                end

                <b>without     :id, [material.id]</b>
                order_by    :score, :desc
            end

</pre>